bliss/models/prior.py

```diff
@@ -271,3 +303,35 @@ def _sample_galaxy_params(self, galaxy_bools):
             s=max_sources,
         )
         return galaxy_params * galaxy_bools
+
+    def _sample_lens_params(self, lensed_galaxy_bools):
+        """Sample latent galaxy params from GalaxyPrior object."""
+        batch_size, n_tiles_h, n_tiles_w, max_sources, _ = lensed_galaxy_bools.shape
+        generate_lens_shape_tensor = lambda n, dist: dist(
+            batch_size,
+            n_tiles_h,
+            n_tiles_w,
+            max_sources,
+            n,
+            device=lensed_galaxy_bools.device,
+        )
```

Comment:
I would avoid the use of lambda functions in this way (this one in particular takes up ~7 lines) and instead create a private method / function